clients/ui/frontend/src/__tests__/cypress/cypress/tests/mocked/modelVersionArchive.cy.ts

```diff
@@ -0,0 +1,303 @@
+    {
+      path: {
+        modelRegistryName: 'modelregistry-sample',
+        apiVersion: MODEL_REGISTRY_API_VERSION,
+        modelVersionId: 3,
+      },
+    },
+    mockBFFResponse(mockModelVersion({ id: '3', name: 'model version 3', state: ModelState.LIVE })),
+  );
+};
+
+describe('Model version archive list', () => {
+  it('No archive versions in the selected registered model', () => {
+    initIntercepts({ modelVersions: [mockModelVersion({ id: '3', name: 'model version 2' })] });
+    modelVersionArchive.visitModelVersionList();
+    verifyRelativeURL('/modelRegistry/modelregistry-sample/registeredModels/1/versions');
+    // TODO: Uncomment when dropdowns are fixed and remove the visit after the comments
+    // modelVersionArchive
+    //   .findModelVersionsTableKebab()
+    //   .findDropdownItem('View archived versions')
+    //   .click();
+    modelVersionArchive.visit();
+    modelVersionArchive.shouldArchiveVersionsEmpty();
+  });
+
+  it('Archived version details browser back button should lead to archived versions table', () => {
+    initIntercepts({});
+    modelVersionArchive.visit();
+    verifyRelativeURL('/modelRegistry/modelregistry-sample/registeredModels/1/versions/archive');
+    modelVersionArchive.findArchiveVersionBreadcrumbItem().contains('Archived version');
+    const archiveVersionRow = modelVersionArchive.getRow('model version 2');
+    archiveVersionRow.findName().contains('model version 2').click();
+    verifyRelativeURL(
+      '/modelRegistry/modelregistry-sample/registeredModels/1/versions/archive/2/details',
+    );
+    cy.go('back');
+    verifyRelativeURL('/modelRegistry/modelregistry-sample/registeredModels/1/versions/archive');
+    modelVersionArchive.findArchiveVersionBreadcrumbItem().contains('Archived version');
+    archiveVersionRow.findName().contains('model version 2').should('exist');
+  });
+
+  it('Archive version list', () => {
+    initIntercepts({});
+    modelVersionArchive.visit();
+    verifyRelativeURL('/modelRegistry/modelregistry-sample/registeredModels/1/versions/archive');
+
+    //breadcrumb
+    modelVersionArchive.findArchiveVersionBreadcrumbItem().contains('Archived version');
+
+    // name, last modified, owner, labels modal
+    modelVersionArchive.findArchiveVersionTable().should('be.visible');
+    modelVersionArchive.findArchiveVersionsTableRows().should('have.length', 2);
+
+    const archiveVersionRow = modelVersionArchive.getRow('model version 1');
+
+    archiveVersionRow.findLabelModalText().contains('5 more');
+    archiveVersionRow.findLabelModalText().click();
+    labelModal.shouldContainsModalLabels([
+      'Financial',
+      'Financial data',
+      'Fraud detection',
+      'Test label',
+      'Machine learning',
+      'Next data to be overflow',
+      'Test label x',
+      'Test label y',
+      'Test label y',
+    ]);
+    labelModal.findCloseModal().click();
+  });
+});
+
+// TODO: Uncomment when we have restoring and archiving mocked
```

Comment:
Am I correct that we are not able to mock this at the moment with Cypress? Both archiving a model or model version as well as restoring them didn't seem to work.